openidc.lua

```diff
@@ -1203,24 +1203,37 @@ local function openidc_access_token(opts, session, try_to_renew)
   return session.data.access_token, err
 end
 
+function openidc_get_path(uri)
```

Comment:
Could we maybe add a short comment here on details/intention?

Reply:
`openidc.lua` is owned by https://github.com/zmartzone/lua-resty-openidc- it's just included here since there is no upstream release that can be pre-installed and must be manually copied.